src/tools/kotlin-schema-field.ts

```diff
@@ -0,0 +1,268 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {SchemaNode} from './schema2graph.js';
```

Comment:
Breaking this out into another file will likely require modification of blaze rules in G3 (depending on if they're using a glob pattern).

Reply:
I've set it up a number of weeks ago with globs() so you can add and remove files without needing to adjust the BUILD file. Yay!